pkg/routeagent_driver/handlers/kubeproxy/vxlan.go

```diff
@@ -301,3 +304,25 @@ func (kp *SyncHandler) createVxLANInterface(activeEndPoint string, ifaceType int
 
 	return nil
 }
+
+func (kp *SyncHandler) ensureLooseModeIsConfigured(iface string) (bool, error) {
+	for i := 0; i < 10; i++ {
+		// Revisit: This is a temporary work-around to fix https://github.com/submariner-io/submariner/issues/2422
+		// Allow the vx-submariner interface to get initialized.
+		time.Sleep(100 * time.Millisecond)
```

Comment:
I've seen that immediately after creating the interface, if we set the rp_filter to 2 and read it, its value sometimes is 2 and later gets updated. Adding this delay seems to allow the interface to get initialised and then updating the rp_filter to 2 was working in the setup I was looking at.

As this is a one time operation (only for GW node) and is not periodically invoked, this small delay should be okay IMO for other platforms.

Reply:
If we do as suggested [here](https://github.com/submariner-io/submariner/issues/2422#issuecomment-1511351483) then it would/should be set to 2 and we'd avoid the delay. But not a big deal - it's negligible anyway.

Reply:
See the later messages...